cmd/image-builder/main.go

```diff
@@ -246,6 +272,20 @@ operating systems like Fedora, CentOS and RHEL with easy customizations support.
 	buildCmd.Flags().String("cache", "/var/cache/image-builder/store", `osbuild directory to cache intermediate build artifacts"`)
 	rootCmd.AddCommand(buildCmd)
 
+	// XXX: add --format=json too?
```

Comment:
I had the same thought and it would mean basically stripping the `YAML` from struct names such as `describeImgYAML` and adding tags for `json. I think that it would make sense and should be fairly easy to do. One can then use `jq` to extract whatever information they need and things become interesting :sweat_smile:

Reply:
Thanks, yeah, that makes sense.

I updated the PR, the test was failing because I forgot to pass --arch in the describe-image smoke test so on non-x86 it would fail. this is fixed now.